src/App.js

```diff
@@ -1,29 +1,29 @@
 import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
 
 import Countries from './pages/Countries';
-import CountriesState from './pages/CountriesState';
+import CountriesStates from './pages/CountriesStates';
 import Cities from './pages/Cities';
 import Posts from './pages/Posts';
 import Login from './pages/Login';
 import ToastMessage from './components/ToastMessage/ToastMessage'
-import Navation from './Navigation';
+import Navigation from './Navigation';
 
-const routeConfig = [
-  {path: '/', element: <Countries />},
-  {path: 'states', element: <CountriesState />},
-  {path: 'cities', element: <Cities />},
-  {path: 'cards', element: <Posts />},
-  {path: 'login', element: <Login />},
+const RouteConfig = [
+  {path: '/', element: <Countries />, label: 'Countries'},
+  {path: '/states', element: <CountriesStates />, label: 'States'},
+  {path: '/cities', element: <Cities />, label: 'Cities'},
+  {path: '/cards', element: <Posts />, label: 'Cards'},
+  {path: '/login', element: <Login />, label: 'Login'},
 ]
 
 const App = () => {
 
   return (
     <Router>
-        <Navation />
+        <Navigation navigationConfig={RouteConfig}/>
```

Comment:
This is a constant, which sense to pass it in props?

Reply:
You pass here `RouteConfig`
What if you have more routes, than you need to render in your menu?